### PR TITLE
169174702 netex variation, netex link to service view, refactoring netex

### PR DESCRIPTION
**Pull request by philosotractor**
# Added
* Variation originally contributed to https://github.com/finnishtransportagency/mmtis-national-access-point/pull/1197, but not merged to master before this
* Netex parts not included to https://github.com/finnishtransportagency/mmtis-national-access-point/pull/1199
   * E.g. netex data-content db field and suppor for exporting/displaying it
   * etc.
* New functionality to top those:
  * service_viewer service card support for netex links
  * Admin panel support for querying netex conversions
* Refactoring of dead code in those parts touched be above mentioned implementations...which bloats this PR unfortunately

### CODE REVIEW

File: ote/src/clj/ote/services/admin.clj [hunk 1]
```diff
@@ -252,11 +253,23 @@
...
+  (mapv (fn [conv]
+          (if (= :ok (keyword (:ote.db.netex/status conv)))
+            (assoc conv :url (export-netex/file-download-url config
+                                                (:ote.db.transport-service/id conv)
+                                                (:ote.db.netex/id conv)))
+            conv))
+        conversions))
+
+(defn- list-netex-conversions-response [config db query]
+  (as->
```
Comment (rckopteri) [inline ote/src/clj/ote/services/admin.clj]: 👍

File: ote/src/clj/ote/services/transport.clj [hunk 2]
```diff
@@ -87,25 +80,37 @@
                 add-error-data))
           modified-services)))
 
+(defn- append-ote-netex-interfaces [{::t-service/keys [id] :as service} config db]
+  (assoc service
+    :ote-interfaces
+    (vec
+      (sort-by :url
+               (mapv #(assoc %
+                        :url (export-netex/file-download-url config id (:ote.db.netex/id %))
```
Comment (rckopteri) [inline ote/src/clj/ote/services/transport.clj]: :ote.db.netex could be changed to ::netex and import ote.db.netex.

File: ote/src/clj/ote/services/transport.clj [hunk 3]
```diff
@@ -136,9 +141,7 @@
 (defn- update-rental-price-classes [service]
   (update-in service [::t-service/rentals ::t-service/vehicle-classes]
              (fn [vehicles]
-               (mapv #(let [before %
-                            after (fix-price-classes % [::t-service/price-classes])]
-                        after)
+               (mapv #(fix-price-classes % [::t-service/price-classes])
```
Comment (rckopteri) [inline ote/src/clj/ote/services/transport.clj]: 👍

File: ote/src/clj/ote/services/transport.clj (hunk 2 above)
Comment (philosotractor) [reply ote/src/clj/ote/services/transport.clj]: Reason for not doing that was a namespace coflict with integration.export.netex

**Pull request merged**